### PR TITLE
Services page

### PR DESCRIPTION
**Pull request by SStrausman**

### CODE REVIEW

**Comment by tjmasterson** [comment]
beautiful!

**Comment by thewildpendulum** [comment]
@SStrausman Fix merge conflicts

**Comment by thewildpendulum** [comment]
@KaraAJC Please review

File: app/controllers/community_resource_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    @community_resources = CommunityResource.all
+  end
+
+  def new
+    @community_resource = CommunityResource.new
+  end
+
+  def create
+    @community_resource = CommunityResource.new(community_resource_params)
+    @community_resource.save
```
Comment (KaraAJC) [inline app/controllers/community_resource_controller.rb:12]: let's keep in mind whether we'll be adding any validations to this model that we'll need to set an if/else statement with errors.

**Comment by KaraAJC** [comment]
works for me !

**Pull request merged**